apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -5,7 +5,7 @@ import ProgressButton from "../progress/ProgressButton";
 
 const styles = {
   section: {
-    width: 970,
+    width: 960,
```

Comment:
Might add a comment along these lines. You might also be able to just leave it at 970, but do `boxSizing: "border-box"`